assets/sass/components/dashboard/_googlesitekit-DashboardPageSpeed.scss

```diff
@@ -161,4 +161,45 @@
 			}
 		}
 	}
+
+	.googlesitekit-pagespeed--recommendations {
+		box-shadow: inset 0 1px 0 rgba(0, 0, 0, 0.1);
+		padding: $grid-gap-phone $grid-gap-phone 0;
+
+		.googlesitekit-pagespeed-recommendations__title {
+			color: $c-black;
+			font-size: 1rem;
+			letter-spacing: 0.1px;
+			line-height: 1.5rem;
+		}
+
+		.googlesitekit-accordion {
```

Comment:
Let's set the color here (for the entire accordion basically) to black too, see the Figma file. Currently it uses the default gray which is hard to read for this content.

Reply:
Agree, updated.